### ISSUE TITLE
Ch 5: Use of subsample of Pew Survey

### ISSUE THREAD

**Issue by mine-cetinkaya-rundel** [OpenIntroOrg/openintro-statistics#8]
I assume this was for simplicity of calculations, which I like, but I feel like the idea of a subsample will be confusing to students who are just starting to learn about drawing inferences from a sample. Also, as a teacher, my first reaction before reading the footnote was "really, exactly 1000 people?". Do you think the benefits of using n = 1000 for simplicity outweigh these potential costs? I'd suggest sticking with the original sample but could be convinced otherwise.

https://github.com/OpenIntroOrg/openintro-statistics/blob/b7e661929a25ca0f7fb530139e1c5aa015f1f459/ch_foundations_for_inf/TeX/ch_foundations_for_inf.tex#L67-L69

**Comment by DavidDiez** [comment]
I took the subsample since otherwise the SE (from sampling variability) gets so small that it doesn't seem believable (e.g. because the SE doesn't account for bias). I had actually tried to write the example out ignoring that aspect and it felt a bit fabricated or that students might not believe it.

If you don't like the exactly 1000 observations (which I have seen happen in actual polls), we can adjust it to a slightly different number (e.g. 1012).

I'm also planning on moving this footnote into a new appendix that gives the details on each of the data sets used rather than keep it prominently listed here, meaning the subsample aspect would become invisible to most readers after the change, reducing the chance of distraction.
